test/studies/adventOfCode/2021/bradc/futures/day18-retNil4.bad

```diff
@@ -1,8 +1 @@
-day18-retNil4.chpl:12: internal error: RES-RES-ION-2818 chpl version 1.26.0 pre-release (c71dc4aa3f)
-Note: This source location is a guess.
-
-Internal errors indicate a bug in the Chapel compiler ("It's us, not you"),
-and we're sorry for the hassle.  We would appreciate your reporting this bug --
-please see https://chapel-lang.org/bugs.html for instructions.  In the meantime,
-the filename + line number above may be useful in working around the issue.
-
+day18-retNil4.chpl:5: error: generic return type 'C?'
```

Comment:
If you make my suggested change to the error message above this line will need to change too.